.gitignore

```diff
@@ -43,3 +43,4 @@ ucp/avatars/
 vendor/
 docker/apache/ssl/
 logs/
+docker/opendkim/keys/
```

Comment:
There's actually a gitignore in the folder, but I think this format makes more sense. I'll amend this branch to remove that gitignore.

Reply:
Those in-folder gitignores does not appear to transfer to me, so each fork would need to add it themselves.